app/javascript/dashboard/components/widgets/conversation/components/GalleryView.vue

```diff
@@ -158,27 +239,130 @@ export default {
         );
       }
     },
+    onClickDownload() {
+      const { file_type: type, data_url: url } = this.activeAttachment;
+      if (!Object.values(ALLOWED_FILE_TYPES).includes(type)) {
+        return;
+      }
+      const link = document.createElement('a');
+      link.href = url;
+      link.download = `attachment.${type}`;
+      link.click();
+    },
   },
 };
 </script>
 <style lang="scss" scoped>
 .gallery-modal--wrap {
-  display: flex;
-  flex-direction: row;
   align-items: center;
-  width: inherit;
+  display: flex;
+  flex-direction: column;
   height: inherit;
+  width: inherit;
+
```

Comment:
Any way we could reduce the css here? We could use `text-block-title` for 14px bold texts, not using flex centers when not needed?